package.json

```diff
@@ -65,7 +65,7 @@
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
-    "extract-text-webpack-plugin": "2.1.0",
+    "extract-text-webpack-plugin": "3.0.0",
```

Comment:
Thanks for linking the issue, didn't know what this was about. 

Verified that it's setting the width to 500px.